src/main/resources/templates/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>hello</title>
+</head>
+<body>
+Hello world
```

Comment:
I think we can remove this html from PR is everything works fine

Reply:
Done